src/components/side-panel/side-panel.tsx

```diff
@@ -33,26 +29,38 @@ interface SidePanelProps {
 class SidePanel extends React.Component<SidePanelProps & WithStyles<CssRules>> {
     render(): ReactElement<any> {
         const { classes, toggleOpen, toggleActive, sidePanelItems, children } = this.props;
-        const { listItemText, leftSidePanelContainer, row, list, icon, projectIconMargin, active, activeArrow, inactiveArrow, arrowTransition, arrowRotate } = classes;
+        const { listItemText, leftSidePanelContainer, row, list, icon, projectIconMargin, active, 
+            iconOpen, iconClose, iconArrow, iconArrowContainer } = classes;
         return (
             <div className={leftSidePanelContainer}>
                 <List>
                     {sidePanelItems.map(it => (
                         <span key={it.name}>
                             <ListItem button className={list} onClick={() => toggleActive(it.id)} onContextMenu={this.handleRowContextMenu(it)}>
                                 <span className={row}>
-                                    {it.openAble ? <i onClick={() => toggleOpen(it.id)} className={`${it.active ? activeArrow : inactiveArrow} 
-                                        ${it.open ? `fas fa-caret-down ${arrowTransition}` : `fas fa-caret-down ${arrowRotate}`}`} /> : null}
+                                    {it.openAble ? (
+                                        <i onClick={() => toggleOpen(it.id)} className={iconArrowContainer}>
+                                            <IconBase icon={IconTypes.PLAY_ARROW} 
+                                                className={`
```

Comment:
We already have `classnames` module in dependencies so we could make use of it here. For example:
```
classnames({
  [iconArrow]: true,
  [active]: it.active,
  [iconOpen]: it.open,
  [iconClose]: !it.close
});
```